src/api/map/routes/map.ts

```diff
@@ -28,6 +38,22 @@ const list = procedure
     }));
 });
 
+const listLandmarkAssessment = procedure
+  .use(authUser())
+  .input(z.object({ 
+    userId: z.string(), 
+    landmark: z.string() 
+  }))
+  .output(z.array(zLandmarkAssessment))
+  .query(async ({ input }) =>
+{
+  return await db.LandmarkAssessment.findAll({
+    where: { userId: input.userId, landmark: input.landmark },
+    attributes: landmarkAssessmentAttributes,
+  }) as LandmarkAssessment[];
```

Comment:
you shouldn't need this `as` type casting.

Reply:
if missing the `as` type casting, it would show an error as `Property 'createdAt' is optional in type 'LandmarkAssessment' but required in type '{ createdAt: Date; score: number; markdown: string | null; }'.`

Reply:
add this comment to the code.